src/Config.hpp

```diff
@@ -15,248 +15,227 @@
  *   along with this program; if not, write to the Free Software
  *   Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307 USA
  *
- * Copyright (C) 2016 EPAM Systems Inc.
+ * Copyright (C) 2017 EPAM Systems Inc.
```

Comment:
2017?

Reply:
This header was copied from displ_be with wrong year. The real real of display manager creating is 2017.